node/cmd/guardiand/node.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/certusone/wormhole/node/pkg/guardiansigner"
 	"github.com/certusone/wormhole/node/pkg/watchers"
 	"github.com/certusone/wormhole/node/pkg/watchers/ibc"
+	"github.com/ethereum/go-ethereum/crypto"
```

Comment:
how did this dependency get in here twice?

Reply:
I guess VSC was doing me a favor. Removed it.